extensions/amp-playbuzz/0.1/amp-playbuzz.css

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.pb_feed_loading {
+        height: 320px;
+        overflow: hidden;
+}
+
+.pb_top_content_container {
+        opacity: 0;
+}
+
+.pb_top_content_container {
+        opacity: 1;
+}
+
+.pb_feed_placeholder_container {
+        position: absolute;
+        top: 0;
+        width: 100%;
+        height: 100%;
+}
+
+.pb_feed_placeholder_container, .pb_feed_placeholder_container {
+        direction: ltr;
+}
+
+.pb_embed_spinner_iframe {
+        position: relative;
+        max-width: 640px;
+        width: 100%;
+        height: 320px;
+        margin: auto;
+        border: 0
+}
+
+.pb_feed_placeholder_inner {
+        position: relative;
+        max-width: 640px;
+        width: 100%;
+        height: 100%;
+        margin: auto;
+}
+
+.pb_feed_placeholder_content {
+        width: 100%;
+        height: 100%;
+        background-color: #f2f2f2;
+        border-radius: 5px;
+}
+
+.pb_feed_placeholder_preloader {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        width: 29px;
+        height: 38px;
+        margin-top: -36px;
+        margin-left: -15px;
+        overflow: hidden;
+        box-sizing: border-box;
+}
+
+.pb_ie_fixer {
```

Comment:
AMP generally allows CSS animations and we can allow it here as well. Let's come back to it in a follow up PR.